.gitignore

```diff
@@ -0,0 +1,5 @@
+*.pdf
+*.gv
+robo_2d.gv
+*.cpp
```

Comment:
You would not want to ignore .cpp files. You can remove this from the .gitignore.